pnpm-lock.yaml

```diff
@@ -859,8 +859,8 @@ packages:
       '@cspell/dict-docker': 1.1.1
       '@cspell/dict-dotnet': 2.0.1
       '@cspell/dict-elixir': 2.0.1
-      '@cspell/dict-en-gb': 1.1.33
       '@cspell/dict-en_us': 2.3.3
+      '@cspell/dict-en-gb': 1.1.33
```

Comment:
Weird change

Reply:
Hopefully it doesn’t keep bouncing order

Reply:
yeah idk what happened, we shall see